roles/openshift_health_checker/openshift_checks/logging/fluentd.py

```diff
@@ -35,9 +35,72 @@ def run(self, tmp, task_vars):
                    "{}".format(check_error))
             return {"failed": True, "changed": False, "msg": msg}
 
+        config_error = self.check_logging_config(fluentd_pods, task_vars)
+        if config_error:
+            msg = ("The following Fluentd logging configuration issue was found:"
+                   "\n-------\n"
+                   "{}".format(config_error))
+            return {"failed": True, "changed": False, "msg": msg}
+
         # TODO(lmeyer): run it all again for the ops cluster
         return {"failed": False, "changed": False, "msg": 'No problems found with Fluentd deployment.'}
 
+    def check_logging_config(self, fluentd_pods, task_vars):
+        """Ensure that the configured Docker logging driver matches fluentd settings.
+        This means that, at least for now, if the following condition is met:
+
+            openshift_logging_fluentd_use_journal == True
+
+        then the value of the configured Docker logging driver should be "journald".
+        Otherwise, the value of the Docker logging driver should be "json-file".
+        Returns an error string if the above condition is not met, or None otherwise."""
+
+        pod_name = fluentd_pods[0]["metadata"]["name"]
+        use_journald = self._exec_oc("exec {} /bin/printenv USE_JOURNAL".format(pod_name), [], task_vars)
```

Comment:
@sosiouxme I decided to read the value of the env var rather than the ansible variable (`openshift_logging_fluentd_use_journal`) since I believe the ansible variable is not set by default (https://github.com/openshift/openshift-ansible/blob/master/inventory/byo/hosts.origin.example).